config/crd/bases/controlplane.cluster.x-k8s.io_k0scontrolplanetemplates.yaml

```diff
@@ -0,0 +1,225 @@
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.11.4
+  labels:
+    cluster.x-k8s.io/v1beta1: v1beta1
+  name: k0scontrolplanetemplates.controlplane.cluster.x-k8s.io
+spec:
+  group: controlplane.cluster.x-k8s.io
+  names:
+    kind: K0sControlPlaneTemplate
+    listKind: K0sControlPlaneTemplateList
+    plural: k0scontrolplanetemplates
+    singular: k0scontrolplanetemplate
+  scope: Namespaced
+  versions:
+  - name: v1beta1
+    schema:
+      openAPIV3Schema:
+        properties:
+          apiVersion:
+            description: 'APIVersion defines the versioned schema of this representation
+              of an object. Servers should convert recognized schemas to the latest
+              internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
```

Comment:
I don't know if it matters, but this syntax will cause the string to become:

```console
APIVersion defines the versioned schema of this representation
              of an object. Servers should convert recognized schemas to the latest
              internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources
```

The `|` syntax would preserve the linefeeds, but would remove the extra indentation:

```suggestion
            description: | 
              APIVersion defines the versioned schema of this representation
              of an object. Servers should convert recognized schemas to the latest
              internal value, and may reject unrecognized values. More info: 
              https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources
```

And the result becomes:

```console
APIVersion defines the versioned schema of this representation
of an object. Servers should convert recognized schemas to the latest
internal value, and may reject unrecognized values. More info: 
https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources
```

The `>` syntax will remove all the linefeeds and indentation and you end up with just a long one-liner:

```suggestion
            description: > 
              APIVersion defines the versioned schema of this representation
              of an object. Servers should convert recognized schemas to the latest
              internal value, and may reject unrecognized values. More info:
              https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources
```

Results in:

```console
APIVersion defines the versioned schema of this representation of an object. Servers should convert recognized schemas to the latest internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources
```

Reply:
It's an autogenerated code, so it will be changed back to the current state if we re-generate it again.